ui/src/components/CoreConcepts/DecrementButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Button} from 'react-bootstrap';
+
+class DecrementButton extends React.PureComponent {
```

Comment:
Have you made these components just to satisfy the task requirements about using `PureComponent`? Because they shouldn't exist otherwise

Reply:
Yep, just to satisfy the task requirements. Otherwise, I would use only functional components.

Reply:
I was talking about, that there is no need for `IncrementButton` / `DecrementButton` components. You could've used `Button` directly inside counter